Trfgen/src/trfgen.c

```diff
@@ -414,6 +428,12 @@ void trfgen_start_test(int test_id, char *host_ip, char *bind_ip, char *host_por
       procIds.pids[procIds.noOfPids++] = pid;
       if(pid == 0)
       {
+         printf("Starting client with bind ip=%s, port=%d\n", bind_ip, port);
+         // Add delay to make sure that ipv6 addr is configured on eth2 i/f
+         if (ip_version == 6) {
+           printf("Sleeping for 5 secs\n");
```

Comment:
Same comment as above on verifying that interface is actually configured.

Reply:
Addressed